app/src/main/java/com/lightricks/ex1/Common.java

```diff
@@ -35,13 +35,15 @@ public static boolean hasReadContactsPermission(Context context) {
     public static void glideSetImage(Context context, ImageView ivImage, String imageName) {
         int imageId = getImageId(context, imageName);
         if (imageId == 0){
+            // NOT thread safe! Think about a safe way to preform this operation (the default image calculation)...
             // Init defultImageId for the first time
             if (defaultImageId == -1){
                 defaultImageId = getImageId(context, "default_image");
             }
             // Validate defaultImageId
             if (defaultImageId == 0){
                 // Oh no! "default_image.jpeg" is missing!
+                // Use logging here! notify about it.
```

Comment:
What's the standard logging system we use in Lightricks?

Reply:
We are using Timber, but you can use android default logging utils..